CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unreleased
+
+* Update contextual breadcrumbs to not show taxonomy breadcrumbs when showing specialist documents  
```

Comment:
Please add the PR number to this line (see other lines for examples).

Reply:
Sorry, PR wasn't created yet when I pushed this.